doc/api/path.md

```diff
@@ -159,19 +165,22 @@ path.dirname('/foo/bar/baz/asdf/quux');
 // Returns: '/foo/bar/baz/asdf'
 ```
 
-A [`TypeError`][] is thrown if `path` is not a string.
+A [`TypeError`][] is thrown if `path` is not a string or URL.
```

Comment:
```suggestion
A [`TypeError`][] is thrown if `path` is not a string or `URL` instance.
```

This might look redundant per se, but there were occasions when users interpret "URL" as any URL (including plain string) rather than `URL` objects, and expect heuristic autodetection from functions taking both paths and URLs.
I think, by making this more explicit in the docs we can lower the possibility of misunderstanding.

Reply:
*(resolved locally)*